projects/demo/src/app/forms/layout-angular/layout-angular.html

```diff
@@ -9,7 +9,13 @@ <h1>Angular - {{layout}} - {{ labelSize ? 'grid' : 'no grid' }}</h1>
 
 <form clrForm [clrLayout]="layout" [clrLabelSize]="labelSize">
   <clr-checkbox-container>
-    <label>Checkbox</label>
+    <label
+      >Checkbox
+      <clr-signpost>
+        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
+        <clr-signpost-content *clrIfOpen> Signpost content </clr-signpost-content>
+      </clr-signpost></label
+    >
```

Comment:
```suggestion
    <label>
      Checkbox
      <clr-signpost>
        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
        <clr-signpost-content *clrIfOpen>Signpost content</clr-signpost-content>
      </clr-signpost>
    </label>
```

unmangle tags and remove extraneous whitespace